docs/pages/database-access/guides/aws-dynamodb.mdx

```diff
@@ -0,0 +1,265 @@
+---
+title: Database Access with AWS DynamoDB
+description: How to access AWS DynamoDB with Teleport Database Access
+---
+
+Access to AWS DynamoDB can be provided by [Teleport Database
+Access](../introduction.mdx) for the AWS Console and API. This allows for
+fine-grain access control through [Teleport's RBAC](../../application-access/introduction.mdx).
+
+This guide will help you to:
+
+- Install the Teleport Database Service.
+- Set up the Teleport Database Service to access the AWS Console and API.
+- Connect to your DynamoDB databases through the Teleport Database Service.
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+![DynamoDB Self-Hosted](../../../img/database-access/guides/aws-dynamodb_selfhosted.png)
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+![DynamoDB Cloud](../../../img/database-access/guides/aws-dynamodb_cloud.png)
+</ScopedBlock>
+
+## Prerequisites
+
+- AWS account with DynamoDB databases.
+- IAM permissions to create IAM roles.
+- `aws` Command Line Interface (CLI) tool installed in PATH.
+- A host, e.g., an EC2 instance, where you will run the Teleport Database Service.
+  This guide assumes an EC2 instance when creating and applying IAM roles, and
+  must be adjusted accordingly for custom configurations.
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+(!docs/pages/includes/tctl.mdx!)
+
+<Admonition type="tip" title="Not yet a Teleport user?">
+If you have not yet deployed the Auth Service and Proxy Service, you should follow
+one of our [getting started guides](../getting-started.mdx) or try our Teleport
+Application Access [interactive learning track](https://play.instruqt.com/teleport/invite/rgvuva4gzkon).
+</Admonition>
+
+We will assume your Teleport cluster is accessible at `teleport.example.com` and
+`*.teleport.example.com`. You can substitute the address of your Teleport Proxy
+Service. (For Teleport Cloud customers, this will be similar to `mytenant.teleport.sh`.)
+
+<Admonition type="note" title="Example Access Control">
+This guide provides an example configuration of IAM access roles as a model,
+and uses an EC2 instance to serve the Teleport Database Access service.
+The level of access provided may not suit your needs, or may not fit your
+organization's access conventions. Adjust as needed to fit your needs.
+</Admonition>
+
+## Step 1/4. Create IAM roles for DynamoDB access
+
+The setup described in this guide requires two IAM roles:
+
+- one to identify the EC2 instance running the Teleport Database Access service,
+- one applied to users that allows the former role to assume it, and allows
+access to DynamoDB.
```

Comment:
I'll leave the structure of this up to you since you prefer the bullets.

I think we should drop the "applied to users" for sure though, and rephrase it similar to "one role that has access to DynamoDB services in AWS".